### ISSUE TITLE
Support for generating Andmore projects

### ISSUE THREAD

**Issue by tobykurien** [ksoichiro/gradle-eclipse-aar-plugin#15]
Since Android ADT plugin is no longer developed by Google, it's been forked into the Andmore project. Andmore's ADT fork uses a different package for the project builders and natures. Here's a typical Andmore `.project` file:

``` xml
<?xml version="1.0" encoding="UTF-8"?>
<projectDescription>
    <name>WebApps</name>
    <comment></comment>
    <projects>
    </projects>
    <buildSpec>
        <buildCommand>
            <name>org.eclipse.andmore.ResourceManagerBuilder</name>
            <arguments>
            </arguments>
        </buildCommand>
        <buildCommand>
            <name>org.eclipse.andmore.PreCompilerBuilder</name>
            <arguments>
            </arguments>
        </buildCommand>
        <buildCommand>
            <name>org.eclipse.jdt.core.javabuilder</name>
            <arguments>
            </arguments>
        </buildCommand>
        <buildCommand>
            <name>org.eclipse.andmore.ApkBuilder</name>
            <arguments>
            </arguments>
        </buildCommand>
    </buildSpec>
    <natures>
        <nature>org.eclipse.andmore.AndroidNature</nature>
        <nature>org.eclipse.jdt.core.javanature</nature>
    </natures>
</projectDescription>
```

It would be great if there was an option for Gradle Eclipse AAR plugin to generate these projects instead of the old ADT version.

**Comment by ksoichiro** [comment]
@tobykurien Thank you for reporting this. I'll look into it.

**Comment by ksoichiro** [comment]
I've added a new option `andmore` to support Andmore.
You can try this with `0.2.0-SNAPSHOT`.

``` gradle
buildscript {
    repositories {
        mavenCentral()
        maven {
            url 'https://oss.sonatype.org/content/repositories/snapshots/'
        }
    }
    dependencies {
        classpath 'com.github.ksoichiro:gradle-eclipse-aar-plugin:0.2.0-SNAPSHOT'
    }
}
```

When `andmore` is set to true, `.classpath` and `.project` will be generated for Andmore.

``` gradle
eclipseAar {
    andmore = true
}
```

**Comment by tobykurien** [comment]
Thank you! I've tested it, and it seems to work. I did notice some weird things though:
- Andmore throws up errors about the output `bin` folder not existing after importing the projects. If I close Eclipse and re-open it, they then compile fine. I think it's a bug in Eclipse as it did this with ADT as well. It might not hurt to create the `bin` folder though.
- The AAR projects are not marked with the Andmore "a" badge, instead they are marked with the Java "j" badge. When I previously converted an ADT project to Andmore, it had the "a" badge. I'm not sure why this is since the `.classpath` and `.project` files have the right content in them (including Andmore nature). It doesn't seem to cause problems though...

**Comment by ksoichiro** [comment]
Thank you for testing and your feedback!
I haven't noticed those points.

> Andmore throws up errors about the output bin folder not existing after importing the projects

Unfortunately, I couldn't have confirmed this issue,
but I'll modify the plugin to create `bin` and `gen` folders if they're not created.
Generating those folders by the plugin will not cause problems, I think.

> The AAR projects are not marked with the Andmore "a" badge, instead they are marked with the Java "j" badge.

I found that it's because the first nature in the generated `.project` file is `org.eclipse.jdt.core.javanature`.
If I change the order of natures, "a" badge is shown; `AndroidNature` should be first.
This also happens with ADT, and creating Android project with ADT also defines `AndroidNature` first,
so I'll change the order for both Andmore and ADT.
Please wait for a while.

**Comment by ksoichiro** [comment]
I've fixed those issues and updated the SNAPSHOT, so you can try it if you have time.

**Comment by tobykurien** [comment]
I've tested it, and it is working flawlessly! Great work :+1: Please feel free to make a release.

**Comment by ksoichiro** [comment]
Thank you for testing! I'll release it soon.

**Comment by ksoichiro** [comment]
I've released v0.2.0.
It'll be synced to Maven Central later.
With `plugins` notation, you can use it right now.

``` gradle
plugins {
  id 'com.github.ksoichiro.eclipse.aar' version '0.2.0'
}
```
